app.py

```diff
@@ -1,46 +1,48 @@
-from flask import Flask, render_template, request, flash, redirect, url_for, jsonify
-from flask_cors import CORS
+from flask import Flask, render_template, request, flash, redirect, url_for, jsonify, g
+# from flask_cors import CORS
 from hashids import Hashids
 import sqlite3
-hashids = Hashids()
+
+
+def get_db_connection():
+    db = sqlite3.connect('database.db')
+    db.row_factory = sqlite3.Row
+    return db
 
 
 
 app = Flask(__name__)
-CORS(app)
+# CORS(app)
 
 app.config['SECRET_KEY'] = 'this should be a secret random string'
 
 hashids = Hashids(min_length=5, salt=app.config['SECRET_KEY'])
-hashid = hashids.encode(8)
-print(hashid)
+# hashid = hashids.encode(8)
+# print(hashid)
 
-def get_db_connection():
-    connection = sqlite3.connect('database.db')
-    connection.row_factory = sqlite3.Row
-    return connection
 
 
-@app.route('/', methods=['GET', 'POST'])
+@app.route('/', methods=('GET', 'POST'))
 def home():
-    connection = get_db_connection()
+    db = get_db_connection()
 
     if request.method == 'POST':
         url = request.form['url']
+
         if not url:
-            flash('The URL is required!')
+            flash('The URL is needed!')
             return redirect(url_for('home'))
-        url_data = connection.execute('INSERT INTO url_db (original_url) VALUES (?)',
+
+        url_data = db.execute('INSERT INTO urls (original_url) VALUES (?)',
                                 (url,))
```

Comment:
You are storing the original url in to the database, but I cant see where the short version is being stored, is it meant to be after that comma? When I look at the database I see the original url I submitted into the form then a column full of 0s.